content/en/docs/_index.md

```diff
@@ -9,11 +9,11 @@ menu:
 
 ## What is Gin?
 
-Gin is a HTTP web framework written in Go (Golang). It features a Martini-like API with much better performance -- up to 40 times faster. If you need smashing performance, get yourself some Gin.
+Gin is a HTTP web framework written in Go (Golang). It features a martini-like API with excellent performance -- up to 40 times faster than other frameworks. If you need smashing performance, get yourself some Gin.
```

Comment:
This is actually more incorrect. Martini is a framework. It is referencing that specifically.

Reply:
Goooootcha, I'll correct that.